### DISCUSSION TITLE
Adding ed25519 ssh keys to ssh/sshd file

### DISCUSSION THREAD

**Discussion by phantoms6** [usnistgov/macos_security#316] [Q&A]
I was wondering if there was any guidance on how to add ed25519 ssh keys to the ssh/sshd files now that it is fips validated as of Feb 2023) according to this: https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-5.pdf

Any help is appreciated! Just not sure where to add it, and the formatting.

**Comment by robertgendler** [comment]
@phantoms6 unfortunately until Apple updates what they're validating and saying is the proper configuration, ed25519 isn't part of the configuration they provide in the man page for `apple_ssh_and_fips` 

This is a related request to Issue #227
